metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_config.py

```diff
@@ -21,6 +21,8 @@
 
 
 class BigQueryUsageConfig(BaseUsageConfig):
+    pydantic_removed_field("query_log_delay")
```

Comment:
This has to get assigned to a method name

```suggestion
    _query_log_delay_removed = pydantic_removed_field("query_log_delay")
```